fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/TempTableInsertPlan.java

```diff
@@ -184,7 +184,7 @@ public int hashCodeWithoutChildren() {
 
     @Override
     public int planHash(@Nonnull final PlanHashMode mode) {
-        return PlanHashable.objectsPlanHash(mode, BASE_HASH, getTempTableReferenceValue());
+        return PlanHashable.objectsPlanHash(mode, BASE_HASH, getChild());
```

Comment:
This should actually use BOTH the reference value and the child. The reference value is safe (to be used for plan hashes; only quantifiers are not).

Reply:
I've added back the `getTempTableReferenceValue()` (and the Boolean `isOwningTemplate`, which I think was an oversight). I'd thought that that was also unstable, but further local tests seem to suggest I was wrong, so either I was comparing the wrong thing or I was comparing this code against hashing the quantifier by mistake. I've re-done the final test (pushing a build to maven local and running the multi-server tests with two JVMs on the latest code), and the plan hashes were stable.